{% extends 'base.html' %}

{% load static %}

{% block body_block %}
    <!-- ==========================================================
         Header banner
    =========================================================== -->
    <div>
        <img class="img-responsive"
             src="{% static 'images/banner.png' %}"
             alt="Banner" width="100%"/>
    </div>

    <!-- ==========================================================
         Partner logos
    =========================================================== -->
    <div class="row" style="margin-top:20px; text-align:center;">
        <div class="col-xs-4 col-sm-3">
            <a href="https://www.polyomics.gla.ac.uk/" target="_blank">
                <img src="{% static 'images/gp.png' %}" alt="gp"
                     class="img-responsive center-block" style="max-height:90px;">
            </a>
        </div>
        <div class="col-xs-4 col-sm-3">
            <a href="https://www.gla.ac.uk/" target="_blank">
                <img src="{% static 'images/uog.png' %}" alt="uog"
                     class="img-responsive center-block" style="max-height:45px;">
            </a>
        </div>
        <div class="col-xs-4 col-sm-3">
            <a href="https://www.wur.nl/en/wageningen-university.htm" target="_blank">
                <img src="{% static 'images/wu.svg' %}" alt="wu"
                     class="img-responsive center-block" style="max-height:45px;">
            </a>
        </div>
    </div>

    <!-- ==========================================================
         Navigation buttons
    =========================================================== -->
    <div class="text-center" style="margin-top:45px; margin-bottom:40px;">
        <a href="/basicviz"
           class="btn btn-default"
           style="opacity:0.8;"
           title="Legacy interface – planned deprecation">
           Go&nbsp;to&nbsp;Experiments&nbsp;(legacy)
        </a>
        <a href="/motifdb" class="btn btn-info">
            Go&nbsp;to&nbsp;MotifDB
        </a>
        <a href="/viz/" class="btn btn-success">
            Try&nbsp;the&nbsp;new&nbsp;Dashboard&nbsp;↗
        </a>
        <a href="https://twitter.com/vdHooft_CompMet" target="_blank"
           class="btn btn-default" style="margin-left:4px;">
           Follow&nbsp;us&nbsp;on&nbsp;X&nbsp;↗
        </a>
    </div>

    <!-- ==========================================================
         People panels
    =========================================================== -->
    <div class="row">
        <div class="col-md-10 col-md-offset-1">

            <!-- Simon Rogers ------------------------------------ -->
            <div class="panel panel-default" style="margin-bottom:15px;">
                <div class="panel-heading"><h4 class="panel-title">Simon Rogers</h4></div>
                <div class="panel-body">
                    <img src="https://avatars2.githubusercontent.com/u/611683?v=3&s=400"
                         alt="srogers"
                         class="img-responsive img-thumbnail"
                         style="width:180px; margin:15px 15px 15px 0; float:left;">
                    Simon is a Senior Technical Consultant (Data and analytics) at
                    <a href="https://www.nss.nhs.scot/" target="_blank">NHS&nbsp;National&nbsp;Services&nbsp;Scotland</a>.
                    He previously worked as a Senior Lecturer at the
                    <a href="http://www.dcs.gla.ac.uk/~srogers/" target="_blank">University&nbsp;of&nbsp;Glasgow</a>,
                    where he made significant contributions to computational metabolomics. During his academic career,
                    Simon pioneered the application of machine learning techniques to metabolomics data, including methods
                    for using dependencies between metabolites to aid in identification, probabilistic methods for metabolite
                    annotation, and the use of peak grouping to improve alignment and differential expression computation.

                    Simon was a key developer of MS2LDA, applying topic modeling for untargeted substructure exploration in
                    metabolomics (published in <a href="https://www.pnas.org/content/113/48/13738.short" target="_blank">PNAS 2016</a>).
                    He co-authored the textbook "<a href="https://www.routledge.com/A-First-Course-in-Machine-Learning/Rogers-Girolami/p/book/9781498738484" target="_blank">A First Course in Machine Learning</a>"
                    with Mark Girolami, now in its second edition. Simon holds a PhD in Machine Learning from the University of Bristol
                    and was an affiliate member of <a href="https://www.polyomics.gla.ac.uk/" target="_blank">Glasgow&nbsp;Polyomics</a>.
                    His expertise includes machine learning and statistical techniques for complex data, particularly in healthcare,
                    metabolomics, and information retrieval. His publications can be found on
                    <a href="https://scholar.google.com/citations?user=fHdAQTIAAAAJ" target="_blank">Google&nbsp;Scholar</a>.
                </div>
            </div>

            <!-- Justin van der Hooft ----------------------------- -->
            <div class="panel panel-default" style="margin-bottom:15px;">
                <div class="panel-heading"><h4 class="panel-title">Justin van der Hooft</h4></div>
                <div class="panel-body">
                    <img src="{% static 'images/justinjjvanderhooft.jpeg' %}"
                         alt="jjvdh"
                         class="img-responsive img-thumbnail"
                         style="width:180px; margin:15px 15px 15px 0; float:left;">
                    Justin is an Assistant Professor in Computational Metabolomics at the Bioinformatics Group at
                    <a href="https://www.wur.nl/en/wageningen-university.htm" target="_blank">Wageningen&nbsp;University</a>,
                    where he leads the <a href="https://vdhooftcompmet.github.io/" target="_blank">Van&nbsp;der&nbsp;Hooft&nbsp;Computational&nbsp;Metabolomics&nbsp;Group</a>.
                    His research vision is to close the gap between what we can see in metabolomics and what we can actually learn from it.
                    His group develops computational metabolomics approaches inspired by natural language processing (NLP) and genomics, including tools like
                    <a href="https://github.com/vdhooftcompmet/MS2LDA" target="_blank">MS2LDA</a>,
                    <a href="https://ms2query.github.io/" target="_blank">MS2Query</a>, and the
                    <a href="https://github.com/matchms/matchms" target="_blank">matchms</a> Python framework.

                    Justin's
                    <a href="https://scholar.google.nl/citations?user=zv9seLwAAAAJ&hl=en&oi=ao" target="_blank">publications</a>
                    include highly-cited papers in Nature Biotechnology and PNAS. He collaborates with researchers from Wageningen University,
                    UCSD, and the Netherlands eScience Center. His group uses the plant root microbiome and human food metabolome as prime
                    applications, as they represent complex metabolite mixtures that, once elucidated, will boost our insights into molecular
                    mechanisms underpinning growth, development, and health.
                </div>
            </div>

            <!-- Joe Wandy ---------------------------------------- -->
            <div class="panel panel-default" style="margin-bottom:15px;">
                <div class="panel-heading"><h4 class="panel-title">Joe Wandy</h4></div>
                <div class="panel-body">
                    <img src="https://avatars2.githubusercontent.com/u/2523391?v=3&u=1084b6ef759d4f6e611da126eab7c5df791703d4&s=400"
                         alt="joe"
                         class="img-responsive img-thumbnail"
                         style="width:180px; margin:15px 15px 15px 0; float:left;">
                    Joe is a Principal Bioinformatician at <a href="https://www.metabolon.com/" target="_blank">Metabolon,&nbsp;Inc.</a>
                    Previously, he was a Data Scientist at <a href="http://www.polyomics.gla.ac.uk/" target="_blank">Glasgow&nbsp;Polyomics</a>
                    where he led the development of <a href="https://github.com/glasgowcompbio/ms2ldaviz" target="_blank">MS2LDAviz</a>
                    and <a href="https://github.com/glasgowcompbio/graphomics" target="_blank">GraphOmics</a>.
                    His <a href="http://theses.gla.ac.uk/7928/" target="_blank">PhD&nbsp;thesis</a> at the University of Glasgow explored Bayesian methods
                    and topic modeling for mass spectrometry data. Joe specializes in developing computational tools for metabolomics and multi-omics data analysis,
                    with expertise in bioinformatics, scientific Python, machine learning, and Bayesian inference.

                    Joe's most influential works include the
                    <a href="https://www.pnas.org/content/113/48/13738.abstract" target="_blank">PNAS 2016 paper</a> on topic modeling for untargeted
                    substructure exploration (386+ citations), <a href="https://www.mdpi.com/2218-1989/9/7/144" target="_blank">MolNetEnhancer</a>
                    (Metabolites 2019, 370+ citations), and the <a href="https://academic.oup.com/bioinformatics/article/34/2/317/4158166" target="_blank">MS2LDA.org web application</a>
                    (Bioinformatics 2018, 126+ citations). He has collaborated with researchers from the University of Glasgow and Wageningen University
                    on tools for metabolomics data analysis. His publications can be found on <a href="https://scholar.google.com/citations?user=K6nz3aoAAAAJ" target="_blank">Google&nbsp;Scholar</a>.
                </div>
            </div>

            <!-- Other contributors ------------------------------- -->
            <div class="panel panel-default" style="margin-bottom:15px;">
                <div class="panel-heading"><h4 class="panel-title">Other contributors</h4></div>
                <div class="panel-body">
                    <p>
                        The following people have contributed code, patches and new features&mdash;MS2LDA would not be
                        where it is today without their help:
                    </p>
                    <ul>
                        <li>
                            <a href="https://scholar.google.com/citations?user=JWTsNqgAAAAJ&hl=en&oi=ao"
                               target="_blank">Rónán&nbsp;Daly</a> and
                            <a href="https://www.polyomics.gla.ac.uk/" target="_blank">Glasgow&nbsp;Polyomics</a> &mdash;
                            website hosting.
                        </li>
                        <li><a href="https://github.com/n-elie" target="_blank">Nicolas&nbsp;Elie</a> &mdash; porting to Python&nbsp;3.</li>
                        <li><a href="https://www.esciencecenter.nl/team/dr-lars-ridder" target="_blank">Lars&nbsp;Ridder</a> &mdash; MAGMa integration.</li>
                        <li><a href="https://www.esciencecenter.nl/team/stefan-verhoeven-bsc" target="_blank">Stefan&nbsp;Verhoeven</a> &mdash; Gensim integration.</li>
                        <li><a href="https://github.com/ymcdull" target="_blank">Yunfeng&nbsp;(Frank)&nbsp;Zhu</a> &mdash; MS1 analysis module.</li>
                        <li><a href="https://www.researchgate.net/profile/Cher-Ong" target="_blank">Ong&nbsp;Cher&nbsp;Wei</a> &mdash; ClassyFire module.</li>
                        <li><a href="https://github.com/j-a-dietrich" target="_blank">Jonas&nbsp;Dietrich</a> &mdash; core developer of the MS2LDA&nbsp;2.0 modernisation&nbsp;effort (<a href="https://github.com/vdhooftcompmet/MS2LDA" target="_blank">preprint soon</a>).</li>
                        <li><a href="https://github.com/rtlortega" target="_blank">Rosina&nbsp;Torres&nbsp;Ortega</a> &mdash; core developer of the MS2LDA&nbsp;2.0 modernisation&nbsp;effort.</li>
                    </ul>
                </div>
            </div>

        </div>
    </div>
{% endblock %}
